### ISSUE TITLE
not smooth when editing 100k size file

### ISSUE THREAD

**Issue by rhinoxi** [NvChad/NvChad#1415]
**Describe the bug**
sample file: [value.zip](https://github.com/NvChad/NvChad/files/9240752/value.zip)
with no other plugin, editing this file is not smooth enough.
and I tried with lunarvim, no such issue.

**Screenshots**
![record](https://user-images.githubusercontent.com/4988151/182340259-9dc63ce7-c1f7-4be9-a773-f00aee633ab1.mp4)

**Desktop (please complete the following information):**
 - Operating System: macOS
 - Terminal: kitty
 - Version of Neovim: 0.7.2

**Comment by siduck** [comment]
go to default lspconfig and remove this if condition and check again 

![image](https://user-images.githubusercontent.com/59060246/182364558-e0b07d88-4c55-495d-af36-a8680b675c89.png)

**Comment by rhinoxi** [comment]
unfortunately not working, even if I disabled the lsp plugin totally.
I tried to disable all plugins one by one, and after the treesitter is disabled, the editing becomes normal...

**Comment by siduck** [comment]
dang :exploding_head: , share that 100k file here so i can test

**Comment by rhinoxi** [comment]
here, sample file: value.zip
thanks 🫡

> **Describe the bug** sample file: [value.zip](https://github.com/NvChad/NvChad/files/9240752/value.zip) with no other plugin, editing this file is not smooth enough. and I tried with lunarvim, no such issue.
> 
> **Screenshots** https://user-images.githubusercontent.com/4988151/182340259-9dc63ce7-c1f7-4be9-a773-f00aee633ab1.mp4
> 
> **Desktop (please complete the following information):**
> 
> * Operating System: macOS
> * Terminal: kitty
> * Version of Neovim: 0.7.2

**Comment by siduck** [comment]
https://user-images.githubusercontent.com/59060246/182407376-bd805dfc-305b-4f71-9350-5dbf704b7460.mp4

both lag for me

**Comment by rhinoxi** [comment]
yes, I just updated lunarvim to 1.1.4 which is released yesterday, and found the same lag too.
so I downgraded the treesitter version to this commit: "fd92e70c69330dd8f2f6753d3d987c34e7dacd24" which is used by lunarvim 1.1.3, and the lag disapppears.
I'll close this.

**Comment by tomasky** [comment]
Is it just   golang, or is it all the other languages as well?

**Comment by rhinoxi** [comment]
currently by setting:
```
require("vim.treesitter.query").set_query("go", "injections", "(comment) @comment")
```
can get rid of the lag
